src/main/java/com/pdffiller/client/dto/FillRequestBody.java

```diff
@@ -11,23 +13,37 @@
    private List<NotificationEmail> notification_emails = new ArrayList<NotificationEmail>();
    private AccessEnum access = AccessEnum.FULL;
 
-   public enum AccessEnum {
-	 FULL("full"),
-	 SIGNATURE("signature");
-	 private String value;
-	 AccessEnum(String value) {
-	   this.value = value;
-	 }
+    public FillRequestBody(
+            Integer document_id,
+            List<NotificationEmail> notification_emails) {
+        this.document_id = document_id;
+        this.notification_emails = notification_emails;
+    }
+
+    public FillRequestBody() {}
+
+    public enum AccessEnum {
+        @SerializedName("full")
```

Comment:
\@Serialized name is unnecessary, but OK. 
Could just use:  private String access = AccessEnum.FULL.value;